src/njoy21/input/GROUPR/Card2/Nsigz.hpp

```diff
@@ -0,0 +1,13 @@
+struct Nsigz {
+  using Value_t = int;
+  static std::string name(){ return "nsigz"; }
+  static std::string description(){
+    return
+      "The nsigz argument specifies the number sigma zeros.\n"
+      "\n"
+      "The sigma zero values are given on Card5.";
+  }
+
+  static int defaultValue(){ return 1; }
+  static bool verify( Value_t v ){ return v >= 0; }
```

Comment:
The sigz array is allocated to Nsigz size. sigz(1) is [always](https://github.com/njoy/NJOY2016/blob/d75b99e22e5e9f0cea98ddb250d3b5493b7ed8ca/src/groupr.f90#L1036) written to. Consequently, nsigz > 0